src/CMakeLists.txt

```diff
@@ -153,14 +153,18 @@ else ()
 endif (PGINDENT)
 
 # Configuration for running pgindent
-if (OBJDUMP AND PGINDENT)
-  add_custom_command(OUTPUT typedefs.list
-    DEPENDS ${PROJECT_NAME}
-    COMMAND sh ${CMAKE_BINARY_DIR}/scripts/generate_typedefs.sh > typedefs.list)
-  add_custom_target(pgindent
-    COMMAND ${PGINDENT} -typedefs typedefs.list -code-base ${CMAKE_SOURCE_DIR}/src
-    DEPENDS typedefs.list)
-endif (OBJDUMP AND PGINDENT)
+if (PGINDENT)
+  if (OBJDUMP)
+    add_custom_command(OUTPUT typedefs.list
+      DEPENDS ${PROJECT_NAME}
+      COMMAND sh ${CMAKE_BINARY_DIR}/scripts/generate_typedefs.sh > typedefs.list)
+    add_custom_target(pgindent
+      COMMAND ${PGINDENT} -typedefs typedefs.list -code-base ${CMAKE_SOURCE_DIR}/src
+      DEPENDS typedefs.list)
+  else()
+	message(STATUS "Found pgindent, but did not have objdump installed. Cannot use pgindent")
```

Comment:
"Found pgindent, but objdump not installed - cannot use pgindent." 

Reads a bit better to me anyway.

Reply:
Or perhaps to not bury the lede:
"Cannot use pgindent - found pgindent, but objdump not installed"

Reply:
Like the second. Done.